po/fr.po

```diff
@@ -0,0 +1,296 @@
+# French translation for Ladybird.
+msgid ""
+msgstr ""
+"Project-Id-Version: ladybird wip\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-09-24 23:32+0300\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#. Translators: this is a format string for the zoom-percent label in the main menu.
+#. For most languages, it doesn't need translating.
+#: Window.cpp:539
+#, c-format
+msgid "%d%%"
+msgstr ""
+
+#: Window.cpp:445
+msgid "A script closed the web page"
+msgstr "Un script a fermé la page web"
+
+#: window.blp:205 window.blp:306
+msgid "Bookmarks"
+msgstr "Marque-pages"
+
+#: window.blp:51
+msgid "Bookmarks & History"
+msgstr "Marque-pages & Historique"
+
+#: Tab.cpp:215 Tab.cpp:239
+msgid "Cancel"
+msgstr "Annuler"
+
+#: window.blp:37
+msgid "Close _Other Tabs"
+msgstr "Fermer _les autres onglets"
+
+#: shortcuts-dialog.blp:63
+msgid "Close current tab"
+msgstr "Fermer l‘onglet actif"
+
+#: Tab.cpp:216
+msgid "Confirm"
+msgstr "Confirmer"
+
+#: shortcuts-dialog.blp:97 shortcuts-dialog.blp:99
+#, fuzzy
```

Comment:
Please don't leave "fuzzy" entries! That is an automatically added label, which means that the string has been altered — slightly — but the (Swedish I presume) translation hasn't been updated yet (but still considered close enough). Since you're making a fresh translation, there should not be any fuzzy entries left.

You can just delete this line that says "fuzzy".

That being said, I assume "Outils de développement" means "Development tools" (utilities), not develop**er** tools? Is that an oversight, or is that how it's commonly said in French?